design/treelist/chaperone.rhm

```diff
@@ -0,0 +1,40 @@
+#lang rhombus
+
+import:
+  lib("rhombus/private/treelist.rkt") open
+
+def orig = #{treelist}(1, 2, 3, -4)
+def tl = #{chaperone-treelist}(orig,
```

Comment:
Can you reformat this call a little bit?

Reply:
I think this start at a test suite is destined for Racket, so I didn't change this, so far.